test/aeso_calldata_tests.erl

```diff
@@ -0,0 +1,75 @@
+%%% -*- erlang-indent-level:4; indent-tabs-mode: nil -*-
```

Comment:
We should have a test that actually creates calldata for something with arguments :-)

Reply:
Yes, done that now

Reply:
Where?

Reply:
I don't see the tests. You mean done, but not pushed?